.eslintrc

```diff
@@ -3,7 +3,7 @@
     "eslint:recommended"
   ],
   "parserOptions": {
-    "ecmaVersion": 2020,
+    "ecmaVersion": 13,
```

Comment:
I couldn't find `13` in the documentation. We can use `latest` to use the latest environment parser in ESLint.
Ref: https://eslint.org/docs/latest/use/configure/language-options#specifying-environments

Reply:
`13` is present in inline documentation, which is same as setting it to `2022`.

![image](https://github.com/nightwatchjs/nightwatch/assets/39924567/ec8d2cf2-3f90-4577-a322-06d426f5f7b0)

I didn't specify `latest` because earlier also `2020` version was mentioned instead of `latest` and I'm not sure of the reason for that, but I suppose it's so that any changes in ESLint do not affect us, as long as we don't explicitly decide to move to a new version ourselves.

Reply:
@beatfactor Can you please look into this once, if it is fine to use `latest` or should we just keep to a specific version?

Also, please review and merge this PR afterwards, as it is a blocker for migrating the `utils/logger` module.